<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>870</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>870</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.494318 rgba(9, 62, 117, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 90 70pt &quot;Bahnschrift Condensed&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>OpenBill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectButtom">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>780</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(68, 68, 68, 255), stop:0.494318 rgba(9, 62, 117, 255), stop:0.98 rgba(68, 68, 68, 255), stop:1 rgba(0, 0, 0, 0));
font: 90 15pt &quot;Bahnschrift Condensed&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>Seleccionar Imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>780</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(68, 68, 68, 255), stop:0.494318 rgba(9, 62, 117, 255), stop:0.98 rgba(68, 68, 68, 255), stop:1 rgba(0, 0, 0, 0));
font: 90 15pt &quot;Bahnschrift Condensed&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
